static/scss/letter.scss

```diff
@@ -0,0 +1,36 @@
+.letter {
+  background: white;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 50px 100px;
+
+  .program-title {
+    margin-top: 30px;
+  }
+
+  .letter-logo {
+    float: right;
+
+    .mit-logo {
+      width: 60px;
+      padding-right: 5px;
+      border-right: 1px solid $medgray;
+    }
+  }
+
+  .letter-content {
+    margin-top: 50px;
+  }
+
+  .signatories {
+    .signatory {
+      margin-right: 15px;
+      display: inline-block;
+    }
+
+    .sig-image {
+      height: 100px;
```

Comment:
Is this necessary since you're already adjusting the size of the image in the template with the `image` templatetag?

Reply:
Just to make sure that even if images have different sizes, they should render inline with each other.